src/index.html

```diff
@@ -1,12 +1,319 @@
+<!-- #region Head-->
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <!-- #endregion -->
+  <body id="home" class="page__body">
+
+    <header class="header">
+
+      <a href="#home">
+        <img 
+          src="/images/logo.png" 
+          alt="logo"
+          class="image"
+        >
+      </a>
+
+      <nav class="nav">
+
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="https://google.com"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+
+        </ul>
+
+      </nav>
+
+    </header>
+
+    <main class="main">
+
+      <section class="product-cards">
+
+        <div class="product-card" data-qa="card">
```

Comment:
```suggestion
    <main class="main">
      <section class="product-cards">
        <div class="product-card" data-qa="card">
```
It is not critical, but anyway, it affects code readability a bit: blank lines between parent and child tags are redundant. But it's good to add one between **sibling** elements, for example between cards, or separate parts inside cards, etc.